ui/widgets/nuxeo-select.js

```diff
@@ -77,9 +77,9 @@ import { IronResizableBehavior } from '@polymer/iron-resizable-behavior/iron-res
           }
 
           paper-dropdown-menu {
-            padding-top: 5px;
+            padding-top: var(--paper-dropdown-menu-padding-top, 5px);
             --paper-input-container: {
-              padding-top: 0;
+              padding: var(--paper-input-container-padding, 0 0 8px 0);
```

Comment:
Are these variables already used by paper-dropdown-menu and paper-input, or are they new? If the are new, I think they should be named after nuxeo-select, since it would be the only element affected by them.

Reply:
They are new, let me see what I can do.

Reply:
This concern is addressed on the [PR for the `maintenance-3.0.x` branch](https://github.com/nuxeo/nuxeo-elements/pull/460).